tools/ColorTool/signing/SignConfig.xml

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<SignConfigXML>
+  <job platform="anycpu" configuration="release" dest="__RELBINPATH__\..\..\..\s\signed" jobname="Console ColorTool" approvers="miniksa;migrie;duhowett">

```

Comment:
I guess I don't need to approve anything....

Reply:
Oh, I can throw yours in there. sec.